apps/editing-toolkit/editing-toolkit-plugin/common/index.scss

```diff
@@ -1,12 +1,34 @@
 body.hide-homepage-title {
-
 	// Allow homepage title to be edited even when hidden
 	// Lighter color to signify not visible from front page
 	.editor-post-title {
-		opacity: .4;
+		opacity: 0.4;
 	}
 }
 
-.interface-interface-skeleton__editor {
-	max-width: 100%;
+body.slider-width-workaround {
+	.interface-interface-skeleton__editor {
+		max-width: 100%;
+	}
+}
+
+@mixin font-smoothing-antialiased {
```

Comment:
Non-blocking comment: would it make for sense to use a `%placeholder` instead of defining a mixin?

Reply:
TIL about SCSS placeholders! The mixin takes no parameters so yeah probably should be a placeholder from what I've learnt about them so far.

Reply:
Feel free to include this change when anti-aliasing is open for all.